hazelcast/src/main/java/com/hazelcast/internal/ascii/CommandParser.java

```diff
@@ -16,8 +16,13 @@
 
 package com.hazelcast.internal.ascii;
 
+import com.hazelcast.nio.Connection;
 import com.hazelcast.nio.ascii.TextDecoder;
 
 public interface CommandParser  {
     TextCommand parser(TextDecoder decoder, String cmd, int space);
```

Comment:
As far as I can see, this old method is not used anymore (except in this interface) - wouldn't it be better to simply remove it?

Reply:
I think it will be a good candidate for removal within the REST authentication unification task. But for now, this was the simplest way to only change the HTTP POST command processing.

Reply:
Fair enough.